libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -1265,6 +1268,7 @@ int HTTPClient::handleHeaderResponse()
             DEBUG_HTTPCLIENT("[HTTP-Client][handleHeaderResponse] RX: '%s'\n", headerLine.c_str());
 
             if(headerLine.startsWith("HTTP/1.")) {
+                _canReuse = (headerLine[sizeof "HTTP/1."] != '0');
```

Comment:
(sizeof "HTTP/1.") == 8, is it what is expected ?

Reply:
(sizeof "HTTP/1.") must be **7**, although I did not check it. I can add the if (_canReuse) but I am not shure what should be done if HTTP/1.0 is requested and HTTP/1.1 is offered.

Reply:
It is 8 (last '\0' is always part of a `"string"`)
If HTTP/1.0 is requested, I think we should just comply and set reuse to false.

Reply:
@d-a-v You are right, I update the PR next week